### ISSUE TITLE
Create Local weather provider module

### ISSUE THREAD

**Issue by diegopau** [oceanbuilders/ob-flutter-app#135]
Create local weather provider module to handle the apis of weather flow station, device, device observation and accumulate them to get datas

**Comment by Ray-LZ** [comment]
Hi @godslave05 I can switch to the local provider and the data displayed changes, are we displaying fake data here? why it's displayed with an exclamation mark?
![image](https://user-images.githubusercontent.com/46169036/98743702-19268500-2376-11eb-9e93-42b090bab82b.png)

**Comment by godslave05** [comment]
Yes the red mark should be removed and should only visible when could not connect to that source

**Comment by godslave05** [comment]
Resolved in 0.7+10
